.travis.yml

```diff
@@ -287,28 +288,39 @@ before_install:
     - cd $TRAVIS_SRC
     - git clone --depth 10 https://github.com/ParRes/Kernels.git PRK
     - echo -e "SHMEMCC=oshcc -std=c99\nSHMEMTOP=$$TRAVIS_INSTALL\n" > PRK/common/make.defs
+    ## Build Libevent
+    - cd $TRAVIS_SRC
+    - wget https://github.com/libevent/libevent/releases/download/release-2.1.10-stable/libevent-2.1.10-stable.tar.gz
+    - tar -xzvf libevent-2.1.10-stable.tar.gz
+    - cd libevent-2.1.10-stable
+    - ./autogen.sh
+    - ./configure --prefix=$PWD/install
+    - make clean all install
     ## Build PMIx
     - >
       if [[ $SOS_BUILD_OPTS = *"with-pmix"* ]]; then
         cd $TRAVIS_SRC
         git clone --depth 10 https://github.com/pmix/pmix pmix
+        git checkout e4a27268
```

Comment:
Is this necessary or just for safety?

Reply:
Yeah, more for stability.  We could take it out if you prefer, or leave a TODO to take the next PRRTE/PMIx tag combo that works in Xenial.  There isn't a tag on PRRTE we can use at the moment.

Reply:
No, I am ok to keep it. May be we can leave a comment here so that we remember to check it out later if we need to.

Reply:
Comment here would be helpful.